### PR TITLE
feat(components/TabBar): add TabBar component

### PR DESCRIPTION
**Pull request by mtikhinskaya**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

No components for Tabs

**What is the new behavior?**

Added a component for Tab and TabBar

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:
Frontify: http://guidelines.talend.com/document/92132#/navigation-layout/tabs

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

File: packages/components/src/TabBar/TabBar.scss [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	:global(.tc-tab-bar-action) {
+		color: tint($black, 40);
+
+		&:hover,
+		&:focus {
+			background-color: tint($brand-primary, 85);
+		}
+
+		&:global(.active) {
+			background-color: tint($brand-primary, 80);
```
Comment (jsomsanith) [inline packages/components/src/TabBar/TabBar.scss]: guideline : `background: theme color with 0.25 opacity, ` --> `tint($brand-primary, 75)` would be closer
Don't forget to change the hover/focus below to 70

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://370.talend.surge.sh)

**Pull request merged**